libkbfs/folder_branch_ops.go

```diff
@@ -551,7 +551,8 @@ func (fbo *folderBranchOps) setHeadLocked(
 		// we know about.  However, if journaling is enabled, and this
 		// MD was written by this device, it may not be merged yet, so
 		// let onMDFlush take care of setting it in that case.
-		if !journalEnabled || key.KID() != md.LastModifyingWriterKID() {
+		if !journalEnabled || key.KID() != md.LastModifyingWriterKID() ||
```

Comment:
D'oh thanks.  And actually, I think this is wrong, because the first head could be coming from the journal as well after a restart.  Arg.  Need to figure out a way to know if this is really merged or not.